### ISSUE TITLE
False positive GPL

### ISSUE THREAD

**Issue by sebathomas** [nexB/scancode-toolkit#2428]
For the file https://github.com/zlib-ng/minizip-ng/blob/master/mz_compat.c, ScanCode wrongly identifies a GPL license based on two lines in the header.

```
"licenses": [
        {
          "key": "gpl-2.0-plus",
          "score": 59.09,
          "name": "GNU General Public License 2.0 or later",
          "short_name": "GPL 2.0 or later",
          "category": "Copyleft",
          "is_exception": false,
          "owner": "Free Software Foundation (FSF)",
          "homepage_url": "http://www.gnu.org/licenses/old-licenses/gpl-2.0-standalone.html",
          "text_url": "http://www.gnu.org/licenses/old-licenses/gpl-2.0-standalone.html",
          "reference_url": "https://scancode-licensedb.aboutcode.org/gpl-2.0-plus",
          "scancode_text_url": "https://github.com/nexB/scancode-toolkit/tree/develop/src/licensedcode/data/licenses/gpl-2.0-plus.LICENSE",
          "scancode_data_url": "https://github.com/nexB/scancode-toolkit/tree/develop/src/licensedcode/data/licenses/gpl-2.0-plus.yml",
          "spdx_license_key": "GPL-2.0-or-later",
          "spdx_url": "https://spdx.org/licenses/GPL-2.0-or-later",
          "start_line": 9,
          "end_line": 10,
          "matched_rule": {
            "identifier": "gpl-2.0-plus_641.RULE",
            "license_expression": "gpl-2.0-plus",
            "licenses": [
              "gpl-2.0-plus"
            ],
            "is_license_text": false,
            "is_license_notice": true,
            "is_license_reference": false,
            "is_license_tag": false,
            "is_license_intro": false,
            "matcher": "3-seq",
            "rule_length": 22,
            "matched_length": 13,
            "match_coverage": 59.09,
            "rule_relevance": 100
          },
          "matched_text": "distributed under the terms of the [same] license [as] [zlib].\n   See the accompanying LICENSE file for"
        }
      ]
```

**Comment by pombredanne** [comment]
@sebathomas Thanks!
Let me push a fix shortly.

**Comment by pombredanne** [comment]
The notice:
>  This program is distributed under the terms of the same license as **zlib.**
>   See the accompanying LICENSE file for the full text of the license.

exists for several licenses but we did not have the variant with the "zlib" word.

In https://github.com/nexB/scancode-toolkit/commit/24904c90091581ee33ff3d6cbcefe007a0f573db
1. existing variants for GPL (multiple versions), BSD, ISC, zlib are modified to require a minimum match coverage of 80% of words matched such as https://github.com/nexB/scancode-toolkit/commit/24904c90091581ee33ff3d6cbcefe007a0f573db#diff-9e1add15c1dcf723ddd9d7ef844ee9295a058130ad34f18fb08ab6288ab179f9
2. https://github.com/nexB/scancode-toolkit/commit/24904c90091581ee33ff3d6cbcefe007a0f573db#diff-91bac1e2b47393e3eb2b49b8d8bf0d14a35d46d10524516ec178768e8d1c2d2e is a new added rule proper (this is the minimal fix that would have been needed)
3. while scanning minizip, there are a few other license notices to add as a refinement for proprietary and zlib licenses and they made it in the same commit such as https://github.com/nexB/scancode-toolkit/commit/24904c90091581ee33ff3d6cbcefe007a0f573db#diff-db12cfeff44ac432eb3c0eed955df5241b753da7b6c94dc0cefbb5b9e26e1d0f or https://github.com/nexB/scancode-toolkit/commit/24904c90091581ee33ff3d6cbcefe007a0f573db#diff-9f19d07032ebae3778e71249f73698e784fea323b0a0c7b836bc5b9db05db1b5 or https://github.com/nexB/scancode-toolkit/commit/24904c90091581ee33ff3d6cbcefe007a0f573db#diff-2bcd024e0505cb8e7521f1435e21e56496792ed42093ff8854ba51cdd663036c

**Comment by pombredanne** [comment]
This has been merged in develop and is in the v21.3.31 release.
Thank you ++
